README.md

```diff
@@ -697,6 +697,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [gopm](https://github.com/gpmgo/gopm) - Go Package Manager
 * [govendor](https://github.com/kardianos/govendor) - Go Package Manager. Go vendor tool that works with the standard vendor file.
 * [gpm](https://github.com/pote/gpm) - Barebones dependency manager for Go.
+* [gvt](https://github.com/FiloSottile/gvt) - `gvt` is a simple vendoring tool made for Go native vendoring (aka GO15VENDOREXPERIMENT), based on gb-vendor. [gvt godoc](http://godoc.org/github.com/FiloSottile/gvt)
```

Comment:
Oh, the template asks you to provide the godoc link inside the PR you're creating.

Thanks. You can also just update this PR, there's no need to make a separate one. But either way works.